.bcr/config.yml

```diff
@@ -0,0 +1,5 @@
+# See https://github.com/bazel-contrib/publish-to-bcr#a-note-on-release-automation
+# for guidance about this section:
+fixedReleaser:
```

Comment:
@meteorcloudy @Wyverald   What's our opinion on releaser names?  Do we have a general robot account for auto-PRs?

Reply:
I don't really care, this only shows up in the PR comment (the actual author of the PR itself always seems to be the publish-to-bcr bot)

Reply:
We could make this @bazel-io like rules_java does it: https://github.com/bazelbuild/bazel-central-registry/pull/818